medicai/dataloader/nnunet/preprocessing.py

```
@@ -0,0 +1,618 @@
+
+    if manifest_file is None or not Path(manifest_file).exists():
+        raise ValueError(
+            "manifest_file must be provided for preprocessing. "
+            "Legacy MSD layout is no longer supported."
+        )
+
+    worker_fn = partial(
+        _process_item_helper,
+        output_dir=output_dir,
+        properties_dir=properties_dir,
+        manifest=manifest,
```

Ruff (0.15.9)

[error] Undefined name `manifest`

(F821)